crates/biome_service/src/settings.rs

```diff
@@ -487,8 +487,8 @@ fn to_file_settings(
             included_files: to_matcher(
                 working_directory,
                 config.include.as_ref(),
-                vcs_config_path,
-                gitignore_matches,
+                None,
+                &[],
```

Comment:
This could be proper of differentiating Ignore and Include Matcher. For now I think it is ok to set to None the `gitignore` on the include matcher.

Reply:
Can you please add a comment?